example/dcgan/dcgan.go

```diff
@@ -61,6 +63,45 @@ func discriminator(nc int64, ndf int64) *nn.SequentialModule {
 	)
 }
 
+func weightInit(m nn.IModule) {
+	if strings.Contains(m.Name(), "Conv") {
```

Comment:
This looks a little hacky -- if a field name contains substring "Conv", then consider it an `nn.Module`.

Reply:
The Apply method will be added in the next PR.